src/HttpClientFactory/Http/src/Logging/LoggingHttpMessageHandlerBuilderFilter.cs

```diff
@@ -51,4 +51,4 @@ public Action<HttpMessageHandlerBuilder> Configure(Action<HttpMessageHandlerBuil
             };
         }
     }
-}
```

Comment:
VS did this. Must be important.

Reply:
I will allow it.

I only complain about whitespace in code. I'm agnostic to ending files with blank line or not 😝